.travis.yml

```diff
@@ -43,7 +50,7 @@ script:
   - ./.ci/build.sh
   - rm -rf private/
   - pytest
-  - coala --non-interactive -V
+  - coala --flush-cache --non-interactive -V
```

Comment:
is this really required?

Reply:
I guess so, yes! Otherwise the build was failing unnecessarily if the files weren't changed

Reply:
Even if there are minor changes, like 2-3 lines of code still it was failing. That's why I added this up to re-run the coala on entire project irrespective of cache